src/main/java/seedu/pill/command/StockCheckCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.pill.command;
+
+import seedu.pill.util.ItemMap;
+import seedu.pill.util.Storage;
+
+public class StockCheckCommand extends Command{
+    private final int threshold;
+
+    public StockCheckCommand(String threshold) {
+        this.threshold = Integer.parseInt(threshold);
+    }
```

Comment:
Should this catch an exception? If I recall correctly, Integer.parseInt throws an exception if threshold is not valid.